action_name:
  - move_kinect_action
  - move_head_action
move_kinect_action:
  pitch_step: 0.4
  yaw_step: 0.7
  min_pitch: -1
  min_yaw: -1
  max_pitch: 1
  max_yaw: 1
  time_step: 1
  pitch_command_topic: kinect_pitch_controller/command
  yaw_command_topic: kinect_yaw_controller/command
  sensor_frame: kinect_frame
move_head_action:
  pitch_step: 0.4
  yaw_step: 0.5
  min_pitch: -1
  min_yaw: -1
  max_pitch: 1
  max_yaw: 1
  time_step: 1
  pitch_command_topic: linear_head_pitch_controller/command
  yaw_command_topic: linear_head_yaw_controller/command
  sensor_frame: middle_thermal_link
